.ci/install_kata_image.sh

```diff
@@ -95,6 +97,48 @@ check_not_empty() {
 	fi
 }
 
+build_image_with_dracut() {
+	image_output=${1}
+	os_version=${2}
+	agent_commit=${3}
+
+	check_not_empty "$image_output" "Missing image"
+	check_not_empty "$os_version" "Missing os version"
+	check_not_empty "$agent_commit" "Missing agent commit"
+
+	pushd "${osbuilder_path}" >/dev/null
+
+	if [ "${TEST_INITRD}" == "yes" ]; then
+		local local image_name="kata-containers-initrd.img"
+		local make_target="initrd"
```

Comment:
This looks a bit dodgy - can you move the `local make_target` above the `if/else` test and then set it in both blocks instead of defining the local *in* each block.

Same comment for other blocks in this script.

Reply:
This mimicks existing code in build_image() and I made sure to verify that it's actually correct - but from a style point of view, you're right, it does look weird.  I'll fix it.